src/modules/Game.class.js

```diff
@@ -1,68 +0,0 @@
-'use strict';
```

Comment:
why have you moved this class to `main.js`
That's the reason tests can't run

Please move it back to a separate file and do the following:


// Uncomment the next lines to use your game instance in the browser
// const Game = require('../modules/Game.class');
// const game = new Game();

Reply:
const Game = require('../modules/Game.class');
const game = new Game();

require doesn`t work in browser. Browser send error: main.js:4 Uncaught ReferenceError: require is not defined